lib/get-tts-size.js

```diff
@@ -0,0 +1,14 @@
+const {noopLogger} = require('./utils');
+const debug = require('debug')('jambonz:realtimedb-helpers');
+
+async function getTtsSize(client, logger, pattern = null) {
```

Comment:
I would like to have of getting a count of ALL of the tts keys, or only the count  by a specific account-sid.  I think the key name should allow that right -- if I recall it is tts:{account_sid}:xxxx (?)

Reply:
I put pattern parameter reserved for that case

await (client, logger, 'tts:accountSid:*')